src/auto/configure

```diff
@@ -3452,6 +3452,97 @@ ac_cpp='$CPP $CPPFLAGS'
 ac_compile='$CC -c $CFLAGS $CPPFLAGS conftest.$ac_ext >&5'
 ac_link='$CC -o conftest$ac_exeext $CFLAGS $CPPFLAGS $LDFLAGS conftest.$ac_ext $LIBS >&5'
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
+
+   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for $CC option to accept ISO C89" >&5
```

Comment:
In my understanding; when refer to ISO C, it is C90. C89 is ANSI C.

Reply:
According to [Wikipedia](https://en.wikipedia.org/wiki/ANSI_C#C89), [cppreference](http://en.cppreference.com/w/c/language/history#Standard_C), and [GCC](https://gcc.gnu.org/onlinedocs/gcc-7.3.0/gcc/Standards.html#C-Language) they're essentially different names for the same things.

Reply:
What I (and those documents) mean is this is "ISO C90" or "ANSI C89", not "ISO C89".

Reply:
Oh, you're just talking about the terminology?  I didn't write that.  That's the expansion of the `AC_PROG_CC_C89` macro from src/configure.ac.  If that should be reworded to "ANSI C89" or "ISO C90", it should be fixed in upstream autoconf.

Reply:
Oh.. sorry, this was my misunderstanding..